_posts/2022-02-04-shard-indexing-backpressure-in-opensearch.markdown

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title:  "Shard Indexing Backpressure in OpenSearch"
+authors:
+  - sisurab
+  - sdharms
+  - tkharban
+  - ramaran
+date:   2022-02-04
+categories:
+  - feature
+---
+
+We are excited with the release of Shard Indexing Backpressure feature in OpenSearch version 1.2. In this post we dive into the feature and how it can improve cluster reliability. The Indexing APIs in OpenSearch such as *_bulk* allow~~s~~ you to write data in the cluster, which is distributed across multiple shards, on multiple data nodes. However, at times indexing requests may suffer performance degradation due to a number of reasons including non-optimal cluster configuration, shard strategy, traffic spikes, available node resources and more. These issues are further exacerbated for larger multi-node cluster and indices with many shards. All of these could cause out-of-memory errors, long GC pauses, and reduced throughput, affecting the overall availability of data nodes in addition to degrading performance. This in turn would impact the node's ability to perform useful work. In addition, these node drop events could cascades due to a lack of effective backpressure which puts the entire cluster at the risk.
```

Comment:
* 'We are excited with the release of Shard Indexing Backpressure feature in OpenSearch version 1.2.' I would drop this line. v 1.2 came out nearly 4 months ago and the 'we are excited..." is boiler plate. Maybe in the next sentence, replace 'the feature' with 'Shard Indexing Backpressure'
* '*_bulk*' -> `_bulk`
* `allow~~s~~` -> do you want the 's' to be strike through?
* 'GC' -> 'garbage collection' 
* 'could cascades' -> 'could cascade'